app/views/schools/participants/edit_name.html.erb

```diff
@@ -1,11 +1,16 @@
 <% content_for :before_content, govuk_back_link( text: "Back", href: schools_participant_path(id: @profile)) %>
+
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_for @profile.user, url: schools_participant_update_name_path, method: :put do |f| %>
+      
+    <span class="govuk-caption-xl"><%= @school.name %></span>
+
+      <%= form_for @profile.user, url: schools_participant_update_name_path, method: :put do |f| %>
```

Comment:
It's a nitpick but indentation in the form block. Just helps with readability in the future.

Reply:
Fixed just pushing up